rwengine/src/render/DebugDraw.hpp

```diff
@@ -43,8 +43,8 @@ class DebugDraw final : public btIDebugDraw {
 
     std::vector<GeometryVertex> lines;
     size_t maxlines;
-    GeometryBuffer *lineBuff;
-    DrawBuffer *dbuff;
+    std::unique_ptr<GeometryBuffer> lineBuff = std::make_unique<GeometryBuffer>();
```

Comment:
just wonder do we need dynamic allocation here?

Reply:
Doesn't seem like it, it only lives as long as the class anyway.

Reply:
So it's "to do" later. ;)